docs/how_to/install.rst

```diff
@@ -0,0 +1,145 @@
+############
+Installation
+############
+
+You can install Aldryn News & Blog either on `Aldryn <http://www.aldryn.com>`_
+or by hand into your own project.
+
+
+*********************
+Aldryn Platform Users
```

Comment:
We should include a direct install link. See https://github.com/aldryn/aldryn-events/blob/master/docs/introduction/installation.rst for example.